components/item/ItemHero.vue

```diff
@@ -61,7 +61,7 @@
         return this.$proxyMedia(this.selectedMedia.about, this.identifier);
       },
       rightsStatement() {
-        return this.selectedMedia.rightsStatement;
+        return this.selectedMedia.webResourceEdmRights.def[0];
```

Comment:
There are item pages that don't show, because of an error. I think it has to do with the def here (or the one in downloadEnabled), but haven't looked at it in more detail.

Look at item-new/2048208/c_pca_28759 for example.

Reply:
I've added a ternary operator to fix this